package.json

```diff
@@ -0,0 +1,80 @@
+{
+  "name": "skylark-ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "test": "jest",
+    "test:ci": "yarn test --ci --coverage",
+    "storybook": "start-storybook -p 6006 -s ./public",
+    "build-storybook": "build-storybook -s ./public",
+    "tsc": "tsc",
+    "prettier": "prettier --write .",
+    "prettier:check": "prettier --check .",
+    "vercel:link": "vercel link --project skylark-ui --scope skylark-platform --yes",
+    "vercel:env": "vercel env --environment=preview pull"
+  },
+  "dependencies": {
+    "@fontsource/inter": "^4.5.14",
+    "@fontsource/work-sans": "^4.5.12",
+    "@radix-ui/react-avatar": "^1.0.1",
+    "autoprefixer": "^10.4.13",
+    "clsx": "^1.2.1",
+    "next": "13.0.2",
+    "postcss": "^8.4.18",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-icons": "^4.6.0",
+    "tailwindcss": "^3.2.1",
+    "tailwindcss-radix": "^2.6.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.19.6",
+    "@storybook/addon-actions": "^6.5.13",
+    "@storybook/addon-essentials": "^6.5.13",
+    "@storybook/addon-interactions": "^6.5.13",
+    "@storybook/addon-links": "^6.5.13",
+    "@storybook/addon-postcss": "^2.0.0",
+    "@storybook/builder-webpack5": "^6.5.13",
+    "@storybook/manager-webpack5": "^6.5.13",
+    "@storybook/react": "^6.5.13",
+    "@storybook/testing-library": "^0.0.13",
+    "@testing-library/dom": "^8.19.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@trivago/prettier-plugin-sort-imports": "^3.4.0",
+    "@types/jest": "^29.2.2",
+    "@types/node": "18.11.9",
+    "@types/react": "18.0.25",
+    "@types/react-dom": "18.0.8",
+    "@typescript-eslint/eslint-plugin": "^5.42.0",
+    "babel-jest": "^29.2.2",
+    "babel-loader": "^8.3.0",
+    "chromatic": "^6.11.4",
+    "eslint": "8.26.0",
+    "eslint-config-next": "13.0.2",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-jest": "^27.1.4",
+    "eslint-plugin-storybook": "^0.6.7",
+    "husky": "^8.0.1",
+    "jest": "^29.2.2",
+    "jest-environment-jsdom": "^29.2.2",
+    "jest-junit": "^14.0.1",
+    "lint-staged": "^13.0.3",
+    "prettier": "^2.7.1",
+    "prettier-plugin-tailwindcss": "^0.1.13",
+    "storybook-addon-next-router": "^4.0.1",
+    "ts-jest": "^29.0.3",
+    "typescript": "4.8.4",
+    "util": "^0.12.5"
+  },
+  "resolutions": {
+    "webpack": "^5"
+  },
+  "readme": "ERROR: No README data found!",
```

Comment:
is this expected ?

Reply:
I have no idea where the readme part has come from, removing